### PR TITLE
[BEAM-6158] Remove test_wordcount_without_save_main_session

### PR DESCRIPTION
**Pull request by markflyhigh**
Since there is a workaround in pull/7710 which enable save_main_session
flag in Python3, test_wordcount_without_save_main_session is no longer
needed. Should use [test_wordcount_fnapi_it](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/wordcount_it_test.py) instead for fnapi testing.

Also merge two ValidatesContainer run scripts to one since they can share
many setup and cleanup code.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | ---

See [.test-infra/jenkins/README](../.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by markflyhigh** [comment]
Run Python Dataflow ValidatesContainer

**Comment by markflyhigh** [comment]
+R: @charlesccychen @tvalentyn

**Comment by markflyhigh** [comment]
Run Seed Job

File: sdks/python/container/run_validatescontainer.sh [hunk 1]
```diff
@@ -47,9 +55,15 @@ gcloud -v
 
 # Build the container
 TAG=$(date +%Y%m%d-%H%M%S)
-CONTAINER=us.gcr.io/$PROJECT/$USER/python
-echo "Using container $CONTAINER"
-./gradlew :beam-sdks-python-container:docker -Pdocker-repository-root=us.gcr.io/$PROJECT/$USER -Pdocker-tag=$TAG --info
+if [[ -z $1 ]]; then
```
Comment (tvalentyn) [inline sdks/python/container/run_validatescontainer.sh]: I'd suggest to require explicit parameter both for py2 and py3, and have once condition where based on the parameter we set:
 ```
IMAGE_NAME="python3"  # Use this to create CONTAINER_IMAGE variable.
PYTHON_INTERPRETER="python3.5"   # Use this in virtualenv command.
```

File: sdks/python/apache_beam/examples/wordcount_it_test.py [hunk 2]
```diff
@@ -103,13 +50,6 @@ def test_wordcount_it(self):
   def test_wordcount_fnapi_it(self):
     self._run_wordcount_it(wordcount.run, experiment='beam_fn_api')
 
-  @attr('Py3IT')
```
Comment (tvalentyn) [inline sdks/python/apache_beam/examples/wordcount_it_test.py]: Do we need to do anything to run wordcount on Python 3 using regular wordcount test?

**Comment by markflyhigh** [comment]
Run Python Dataflow ValidatesContainer

File: sdks/python/apache_beam/examples/wordcount_it_test.py (hunk 2 above)
Comment (markflyhigh) [reply sdks/python/apache_beam/examples/wordcount_it_test.py]: I don't think we need to do anything particular. Setup python3 environment and use python3 interpreter should be sufficient.

File: sdks/python/container/run_validatescontainer.sh (hunk 1 above)
Comment (markflyhigh) [reply sdks/python/container/run_validatescontainer.sh]: sg.

File: sdks/python/apache_beam/examples/wordcount_it_test.py (hunk 2 above)
Comment (tvalentyn) [reply sdks/python/apache_beam/examples/wordcount_it_test.py]: So, will we continue to run wordcount tests for Python3 after this change is submitted?

**Comment by markflyhigh** [comment]
If moving script changes to separate PR make things more clear, I can split them. Both of them will touch run scripts but I think they can be reviewed separately.

**Comment by tvalentyn** [comment]
It is also easier to rollback independent changes separately.

File: sdks/python/apache_beam/examples/wordcount_it_test.py (hunk 2 above)
Comment (markflyhigh) [reply sdks/python/apache_beam/examples/wordcount_it_test.py]: yes. I just updated the code. Changes in `run_validatescontainer_py3.sh` should be clear. Basically `ValidatesContainer` will be run.

**Comment by markflyhigh** [comment]
update done. PTAL @tvalentyn

File: sdks/python/container/run_validatescontainer_py3.sh [hunk 3]
```diff
@@ -82,7 +83,7 @@ SDK_LOCATION=$(find dist/apache-beam-*.tar.gz)
 # Run ValidatesRunner tests on Google Cloud Dataflow service
 echo ">>> RUNNING DATAFLOW RUNNER VALIDATESCONTAINER TEST"
 python setup.py nosetests \
-  --attr Py3IT \
+  --attr ValidatesContainer \
```
Comment (tvalentyn) [inline sdks/python/container/run_validatescontainer_py3.sh]: Ok, I totally missed this change in the earlier revision, now it is clear what keeps WordCount running, thanks.

**Comment by tvalentyn** [comment]
Run Python Dataflow ValidatesContainer

**Comment by markflyhigh** [comment]
@charlesccychen Please take a look and merge.

**Comment by charlesccychen** [comment]
Thanks! This LGTM.

**Pull request merged**